templates/api-blueprint.swig

```diff
@@ -102,6 +106,10 @@ FORMAT: 1A
 ### {{ resource.title|default('Resource') }} [{{ resource.href }}]
 
 {{ resource.description }}
```

Comment:
Is it not possible to construct a refract structure where the description is set? Is that invalid by the spec? Right now these can be set by the Swagger adapter, for example.